### ISSUE TITLE
Nonetype for score_breakdown.

### ISSUE THREAD

**Issue by CarlColglazier** [CarlColglazier/cheesecake#9]
```
[2019-03-09 16:50:01,440] ERROR in app: Exception on /api/rankings/2019ncwak [GET]
Traceback (most recent call last):
  File "/home/carl/.local/share/virtualenvs/cheesecake-7w3mno1X/lib/python3.6/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/carl/.local/share/virtualenvs/cheesecake-7w3mno1X/lib/python3.6/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/carl/.local/share/virtualenvs/cheesecake-7w3mno1X/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/home/carl/.local/share/virtualenvs/cheesecake-7w3mno1X/lib/python3.6/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/carl/.local/share/virtualenvs/cheesecake-7w3mno1X/lib/python3.6/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/home/carl/.local/share/virtualenvs/cheesecake-7w3mno1X/lib/python3.6/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/carl/.local/share/virtualenvs/cheesecake-7w3mno1X/lib/python3.6/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/carl/.local/share/virtualenvs/cheesecake-7w3mno1X/lib/python3.6/site-packages/flask_caching/__init__.py", line 686, in decorated_function
    rv = f(*args, **kwargs)
  File "./cheesecake/api/__init__.py", line 179, in get_rankings
    if match.score_breakdown[color]["habDockingRankingPoint"]:
TypeError: 'NoneType' object is not subscriptable
```

**Comment by CarlColglazier** [comment]
Fixed in 1f7ef754d6de693365fa36305e98629961bdeadd

**Comment by CarlColglazier** [comment]
Fixed in 1f7ef754d6de693365fa36305e98629961bdeadd
